src/V3EmitCHeaders.cpp

```diff
@@ -22,6 +22,7 @@
 #include "V3Global.h"
 
 #include <algorithm>
+#include <unordered_set>
```

Comment:
```suggestion
#include <set>
```

unordered set will be slower here

Reply:
Ok, I changed it to `set`.